### PR TITLE
Refactor News section

### PR DESCRIPTION
**Pull request by thebeyondr**
image
- Refactor news items into its own component
- Add `data/news.ts` to hold news items
- Create `NewsInterface` to keep news items consistent
- Add hover styles & animations to news items
- Add animation to main heading
- Refactor responsive text sizes in header and news sections
- Add subtle shadow to nav
- Add heading to news items: "What's Happening"
- Add underline animation for nav items for consistency
- Update SEO for projects & project details
- Update project images
- Update styles on project detail page

### CODE REVIEW

**Comment by ChialiT** [comment]
like very much the animation, great work.

Just not sure about the heading "what's new"

and there is hover state difference now, I think in this component actually work better.
on the top nav one, the hover state is actually change the opacity, but personally I think the underline works better.

File: app/page.tsx [hunk 1]
```diff
@@ -38,105 +44,7 @@ export default function IndexPage() {
         </div>
       </div>
 
-      <div className="flex flex-col bg-white">
-        <div className="flex flex-col justify-between gap-5 border-y border-black px-[32px] py-5 md:flex-row md:gap-0 lg:px-20">
-          <div className="flex flex-col items-start justify-center gap-2 lg:flex-row lg:items-center">
-            <div className="flex w-[280px] items-center justify-start gap-2">
-              <Image src={PSEIcon1} alt="pseicon1" height={16} width={16} />
```
Comment (ChialiT) [inline app/page.tsx:45]: can we do something better with the alt text here, like alt="learn and share"
the icon should name it better as well but I know you are just using it as it is.

Comment (ChialiT) [reply app/page.tsx:45]: i'm terrible on reviewing, i'm viewing the old code, my bad.

File: data/news.ts
Comment (ChialiT) [inline data/news.ts:1]: like this approach way better!!!

**Review by ChialiT — APPROVED** [review]
I don't have much to add, just want to make sure the hover state are the same, maybe it can be in different PR.

**Comment by thebeyondr** [comment]
> on the top nav one, the hover state is actually change the opacity, but personally I think the underline works better.

I agree, can add to this PR no prob!

**Review by samueloh99 — APPROVED** [review]

File: data/news.ts
Comment (thebeyondr) [reply data/news.ts:1]: 🙏🏽

File: app/page.tsx (hunk 1 above)
Comment (thebeyondr) [reply app/page.tsx:45]: The `alt` is meant to be read by assistive technologies like screen readers. The standard is that images that are decorative (like these icons) should have `alt=""` so its ignored by assistive technologies. I'll update accordingly!

**Pull request merged**